src/Solnet.Wallet/Ed25519Bip32.cs

```diff
@@ -12,59 +12,50 @@ namespace Solnet.Wallet
     /// </summary>
     public class Ed25519Bip32
     {
-        private readonly string curve = "ed25519 seed";
-        private readonly uint hardenedOffset = 0x80000000;
-
         /// <summary>
-        /// The seed used for key generation.
+        /// The seed for the Ed25519 BIP32 HMAC-SHA512 master key calculation.
+        /// </summary>
+        private const string Curve = "ed25519 seed";
+        
+        /// <summary>
+        /// 
         /// </summary>
-        private byte[] _seed;
+        private const uint HardenedOffset = 0x80000000;
         
         /// <summary>
         /// The computed master key.
         /// </summary>
-        private byte[] _masterKey;
+        private readonly byte[] _masterKey;
         
         /// <summary>
         /// The computed chain code.
         /// </summary>
-        private byte[] _chainCode;
+        private readonly byte[] _chainCode;
 
         /// <summary>
         /// Initialize the ed25519 based bip32 key generator with the passed seed.
         /// </summary>
         /// <param name="seed">The seed.</param>
         public Ed25519Bip32(byte[] seed)
         {
-            _seed = seed;
-            (_masterKey, _chainCode) = GetMasterKeyFromSeed(_seed);
+            (_masterKey, _chainCode) = GetMasterKeyFromSeed(seed);
         }
 
         /// <summary>
         /// Gets the master key used for key generation from the passed seed.
         /// </summary>
-        /// <param name="seed"></param>
-        /// <returns></returns>
-        public (byte[] Key, byte[] ChainCode) GetMasterKeyFromSeed(byte[] seed)
-        {
-            using (HMACSHA512 hmacSha512 = new HMACSHA512(Encoding.UTF8.GetBytes(curve)))
-            {
-                var i = hmacSha512.ComputeHash(seed);
-
-                var il = i.Slice(0, 32);
-                var ir = i.Slice(32);
-
-                return (Key: il, ChainCode: ir);
-            }
-        }       
+        /// <param name="seed">The seed used to calculate the master key.</param>
+        /// <returns>A tuple consisting of the key and corresponding chain code.</returns>
+        private (byte[] Key, byte[] ChainCode) GetMasterKeyFromSeed(byte[] seed)
+            => HmacSha512(Encoding.UTF8.GetBytes(Curve), seed);
 
         /// <summary>
         /// Computes the child key.
         /// </summary>
         /// <param name="key"></param>
         /// <param name="chainCode"></param>
         /// <param name="index"></param>
```

Comment:
Fix params description

Reply:
Force pushed to merge the cake/pipeline changes in one commit but this has been resolved.